README.md

```diff
@@ -73,40 +73,47 @@ github_release and github_actions_secret in the github_provider!) -->
     cp backend.tfvars-template backend.tfvars
     ${EDITOR} backend.tfvars
     ```
+    Paste in the GUID (into `bucket`) and the `region` of the S3 bucket that holds Terraform state. These should be the same as [used in the notifications-api](https://github.com/GSA/notifications-api/blob/7a1c49b7c70245f59f62d8b0ea4311f6393c4fe4/terraform/sandbox/providers.tf#L11) repo.
 
 1. Copy the `.backend.secrets-template` and edit in the sensitive values for the S3 bucket.
 
     ```bash
     cp .backend.secrets-template .backend.secrets
     ${EDITOR} .backend.secrets
     ```
+    The file must contain credentials for the S3 bucket that holds Terraform state. If you have previously set yourself up to run Terraform in Notify's other repos, then you may have these credentials in your local `~/.aws/credentials` file and you may copy them from there. Otherwise, [use these instructions](https://github.com/GSA/notifications-api/tree/main/terraform#use-bootstrap-credentials).
 
 1. Set a variable with the name of the environment you want to work with
 
     ```bash
     export ENV_NAME=[environment_name]
     ```
+    If you are new to the project, you will probably start with the environment name `development`.
 
-1. Copy the `terraform.tfvars-template` and edit in any variable customizations for the target environment.
+1. You don't need to do this unless you are creating a brand-new environment. If you are doing that, copy the `terraform.tfvars-template` and edit in any variable customizations for the target environment.
 
     ```bash
     cp terraform.tfvars-template terraform.${ENV_NAME}.tfvars
     ${EDITOR} terraform.${ENV_NAME}.tfvars
     ```
 
 1. Copy the `.env.secrets-template` and edit in the values for the Cloud
-   Foundry service account and your AWS deployment user.
+Foundry service account and your AWS deployment user.
 
     ```bash
     cp .env.secrets-template .env.${ENV_NAME}.secrets
     ${EDITOR} .env.${ENV_NAME}.secrets
     ```
+    * `TF_VAR_aws_access_key_id` and `TF_VAR_aws_secret_access_key` are created within the AWS console, associated with an IAM role with appropriate permissions.
+    * `TF_VAR_cf_username` and `password` refer to SpaceDeployer credentials which were output from the `cf service-key` command in the [prerequisites](#prerequisites).
+    * `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are the same as what you saved in `.backend.secrets` in a previous step.
 
 1. Run Terraform init to set up the backend.
 
     ```bash
     docker-compose --env-file .backend.secrets run --rm terraform init -backend-config backend.tfvars
     ```
+    If you get the error `Failed to query available provider packages` that's a Zscaler problem.
```

Comment:
I don't have Zscaler problems.  Do people with Zscaler problems know how to address them in this sort of context?

Reply:
Honestly, how to address Zscaler problems is a bit of an open question. It is too big and contentious a topic to go into in this README.

Reply:
Right, at that point it's a matter of reaching out to others in the organization to troubleshoot.